frontend/tests/emailWorkflow.test.ts

```diff
@@ -95,3 +97,29 @@ test('forgot password', async ({ page, tempUser }) => {
   loginPage = await new LoginPage(page).waitFor();
   await expect(loginPage.page.getByText('The email you clicked has expired')).toBeVisible();
 });
+
+test('register via new-user invitation email', async ({ page }) => {
+  await loginAs(page.request, 'admin', defaultPassword);
+  const adminPage = await new AdminDashboardPage(page).goto();
+  const projectPage = await adminPage.openProject('Sena 3', 'sena-3');
+
+  const uuid = randomUUID();
+
+  const newEmail = `${uuid}@mailinator.com`;
+
+  const addMemberModal = await projectPage.clickAddMember();
+  await addMemberModal.emailField.fill(newEmail);
+  await addMemberModal.selectEditorRole();
+  await addMemberModal.submitButton.click();
+
+  // Check invite link returnTo is relative path, not absolute
+  const inboxPage = await getInbox(page, uuid).goto();
+  const emailPage = await inboxPage.openEmail();
+  const invitationUrl = await emailPage.getFirstLanguageDepotUrl();
+  expect(invitationUrl).not.toBeNull();
+  expect(invitationUrl!).toContain('register');
+  expect(invitationUrl!).toContain('returnTo=')
+  expect(invitationUrl!).not.toContain('returnTo=http')
+
+  // No need to clean up temp user account as user was never created
```

Comment:
If you got this far, surely it must be worth it to test that the link really works:
- That they can register
- That they're in the project they were invited to

Reply:
I would be fine with just clicking the link and landing on the register page with the email prefilled. But it would be good to know that they get added to the project since that's kinda the whole point.

Reply:
Done in commit f4fa561417fb588c131a79b2140bcfa2c0ad79b2. Please check out the PR branch and run `pnpm test emailWorkflow` a few times; I had some flakiness related to Playwright clicking on the maildev link, which I couldn't reproduce consistently. I'd like to know if it was just my machine, or if it's flaky on other dev machines as well. When Playwright did successfully click on the maildev link, the rest of the test passed.

Leaving this conversation unresolved until we can determine if the maildev link flakiness is reproducible or not.